be/test/formats/orc/orc_chunk_reader_test.cpp

```diff
@@ -1100,6 +1100,50 @@ TEST_F(OrcChunkReaderTest, TestReadBinaryColumn) {
     }
 }
 
+/**
+ * ORC format: a:bigint,b:varchar,c:varchar
+ * Data:
+ * {a: 1, b: "123456789012", c: "123456"}
+ * {a: 2, b: "12345678901", c: "12345"}
+ */
+TEST_F(OrcChunkReaderTest, TestReadVarcharColumn) {
+    SlotDesc slot_descs[] = {
+            {"a", TypeDescriptor::from_logical_type(LogicalType::TYPE_BIGINT)},
+            {"b", TypeDescriptor::from_logical_type(LogicalType::TYPE_VARCHAR)},
+            {"c", TypeDescriptor::from_logical_type(LogicalType::TYPE_VARCHAR)},
+            {""},
+    };
+    slot_descs[1].type.len = 6;
+    slot_descs[2].type.len = 6;
+    static const std::string input_orc_file = "./be/test/exec/test_data/orc_scanner/10380.orc";
+    std::vector<SlotDescriptor*> src_slot_descriptors;
+    ObjectPool pool;
+    create_slot_descriptors(_runtime_state.get(), &pool, &src_slot_descriptors, slot_descs);
+
+    {
+        OrcChunkReader reader(_runtime_state->chunk_size(), src_slot_descriptors);
+        reader.set_broker_load_mode(false);
```

Comment:
And why disable broker load mode

Reply:
@Smith-Cruise set_broker_load_mode(false) indicates that the strict mode is disabled.